spec/helpers/transaction_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe TransactionHelper do
+
+  it "runs callbacks after outermost transaction is committed" do
```

Comment:
Add a test verifying that the callback is not run when the transaction is rolled back.